### PR TITLE
Use ROS ENV Variable: ROS_DISTRO

### PR DESCRIPTION
**Pull request by StephanOpfer**

### CODE REVIEW

**Review by dhananjaysathe — APPROVED** [review]

File: alica_ros_turtlesim/README.md [hunk 1]
```diff
@@ -216,7 +216,7 @@ In this file, you need to implement the teleportation of the  turtles:
 follow the standard ros build step.
 ```
 cd catkin_ws
-source /opt/ros/<enter-your-ros-version-name-here>/setup.bash
+source /opt/ros/$ROS_DISTRO/setup.bash
```
Comment (dhananjaysathe) [inline alica_ros_turtlesim/README.md]: sorry yeah this wont be set if sourcing isnt done yet

**Review by veerajsk — APPROVED** [review]

File: alica_ros_turtlesim/README.md (hunk 1 above)
Comment (StephanOpfer) [reply alica_ros_turtlesim/README.md]: okay i fixed that ^^

**Review by abhi1287 — COMMENTED** [review]
Looks hacky to me. Won't this break if you have more than one distro?

**Comment by StephanOpfer** [comment]
> Looks hacky to me. Won't this break if you have more than one distro?

Obviously it will, but compared to before it did not work at all. So I guess that is at least a minor improvement.

**Pull request merged**